packages/kbn-custom-icons/src/components/agent_icon/get_agent_icon.ts

```diff
@@ -66,6 +66,14 @@ const darkAgentIcons: { [key: string]: string } = {
   rust: darkRustIcon,
 };
 
+const getAgentNameWithoutPrefix = (agentName: string) => {
```

Comment:
`getAgentNameWithoutPrefix` the function name is a bit misleading as it not only removes the prefix but also the suffix, right?

Reply:
Totally agree @AlexanderWert , changed to `sanitizeAgentName`